alfresco-solrapi-client/alfresco-solrapi-client-spi/src/main/java/eu/xenit/alfresco/solrapi/client/spi/query/AclsQueryParameters.java

```diff
@@ -0,0 +1,25 @@
+package eu.xenit.alfresco.solrapi.client.spi.query;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonInclude.Include;
+import eu.xenit.alfresco.solrapi.client.spi.dto.AclChangeSet;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Set;
+import lombok.Data;
+import lombok.experimental.Accessors;
+
+@Data
+@Accessors(chain = true)
+@JsonInclude(Include.NON_NULL)
+public class AclsQueryParameters {
+    List<Long> aclChangeSetIds;
+    Long fromId;
+    int maxResults;
+
+    public AclsQueryParameters withAclChangeSets(List<Long> aclChangeSetIds) {
+        this.aclChangeSetIds = aclChangeSetIds;
+        this.maxResults = Integer.MAX_VALUE;
```

Comment:
why is this set here?

Reply:
I removed it. I put it because in getTransactions there is this comment:
    * @param maxResults maximum number of results - use {@code 0} or {@code Integer.MAX_VALUE} in order to omit